docs/recipes/react.md

```diff
@@ -0,0 +1,98 @@
+# Testing React components
+
+This recipe is giving guidelines to test React components.
+
+## Setting up Babel
+
+The first thing you need to do is to set up `babel` to transpile JSX code from the tests. To do that, I'd recommend the [babelrc recipe](https://github.com/sindresorhus/ava/blob/master/docs/recipes/babelrc.md) using `babel-preset-react`. You can also have a look at this sample project configuration: https://github.com/adriantoine/ava-enzyme-demo
```

Comment:
Linkify `babel-preset-react` and linkify `sample project configuration` instead of showing the full URL.